src/components/pages/state/charts/index.js

```diff
@@ -55,22 +66,44 @@ export default ({ history }) => {
 
   return (
     <div className={chartsStyles.chartsContainer}>
-      <BarChart
-        data={getDataForField(data, 'totalTestResultsIncrease')}
-        lineData={dailyAverage(data, 'totalTestResultsIncrease')}
-        fill={totalColor}
-        {...props}
-      />
-      <BarChart
-        data={getDataForField(data, 'positiveIncrease')}
-        fill={positiveColor}
-        {...props}
-      />
-      <BarChart
-        data={getDataForField(data, 'deathIncrease')}
-        fill={deathsBarColor}
-        {...props}
-      />
+      <div>
```

Comment:
After I merge this into the feature branch, I'll update that branch from master to give you access to our new [grid system](https://covid19tracking.github.io/website-docs/?path=/story/layout--grids) to make these charts responsive. I would say we want these as four charts (25% wide) on large breakpoints, two (50%) on medium, and one (full width) on small.

Reply:
lovely, that will be much preferred to the size hard coding that's been done so far.